candidate_relay_test.go

```diff
@@ -22,6 +25,158 @@ func optimisticAuthHandler(string, string, net.Addr) (key []byte, ok bool) {
 	return turn.GenerateAuthKey("username", "pion.ly", "password"), true
 }
 
+func passiveTCPRelayGatherAndExchangeCandidates(agentWithRelay, agentNoRelay *Agent) {
```

Comment:
The other test in this file uses a helper `gatherAndExchangeCandidates`. This is a modified copy of that. Specifically dealing with the trickle problem that comes with pions generation of TCP candidates (and in particularly "active" tcptype candidates).